iotdb-client/jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBAbstractDatabaseMetadata.java

```diff
@@ -0,0 +1,3194 @@
+  private static final String FUNCTION_NAME = "FUNCTION_NAME";
+  private static final String FUNCTION_SCHEM = "FUNCTION_SCHEM";
+  private static final String FUNCTION_TYPE = "FUNCTION_TYPE";
+  private static final String GRANTEE = "GRANTEE";
+  private static final String GRANTOR = "GRANTOR";
+  protected static final String IS_AUTOINCREMENT = "IS_AUTOINCREMENT";
+  private static final String IS_GRANTABLE = "IS_GRANTABLE";
+  protected static final String IS_NULLABLE = "IS_NULLABLE";
+  private static final String KEY_SEQ = "KEY_SEQ";
+  private static final String LENGTH = "LENGTH";
+  protected static final String NULLABLE = "NULLABLE";
+  protected static final String NUM_PREC_RADIX = "NUM_PREC_RADIX";
+  protected static final String ORDINAL_POSITION = "ORDINAL_POSITION";
+  private static final String PK_NAME = "PK_NAME";
+  private static final String PKCOLUMN_NAME = "PKCOLUMN_NAME";
+  private static final String PKTABLE_CAT = "PKTABLE_CAT";
+  private static final String PKTABLE_NAME = "PKTABLE_NAME";
+  private static final String PKTABLE_SCHEM = "PKTABLE_SCHEM";
+  protected static final String PRECISION = "PRECISION";
+  protected static final String PRIMARY = "PRIMARY";
+  private static final String PRIVILEGE = "PRIVILEGE";
+  private static final String PROCEDURE_CAT = "PROCEDURE_CAT";
+  private static final String PROCEDURE_NAME = "PROCEDURE_NAME";
+  private static final String PROCEDURE_SCHEM = "PROCEDURE_SCHEM";
+  private static final String PROCEDURE_TYPE = "PROCEDURE_TYPE";
+  private static final String PSEUDO_COLUMN = "PSEUDO_COLUMN";
+  private static final String RADIX = "RADIX";
+  protected static final String REMARKS = "REMARKS";
+  private static final String SCALE = "SCALE";
+  private static final String SCOPE = "SCOPE";
+  private static final String SPECIFIC_NAME = "SPECIFIC_NAME";
+  protected static final String SQL_DATA_TYPE = "SQL_DATA_TYPE";
+  protected static final String SQL_DATETIME_SUB = "SQL_DATETIME_SUB";
+  protected static final String TABLE_CAT = "TABLE_CAT";
+  protected static final String TABLE_SCHEM = "TABLE_SCHEM";
+  protected static final String TABLE_NAME = "TABLE_NAME";
+  protected static final String TABLE_TYPE = "TABLE_TYPE";
+  protected static final String TYPE_CAT = "TYPE_CAT";
+  protected static final String TYPE_NAME = "TYPE_NAME";
+  protected static final String TYPE_SCHEM = "TYPE_SCHEM";
+  private static final String UPDATE_RULE = "UPDATE_RULE";
+
+  private static final String SHOW_FUNCTIONS = "show functions";
+  private static final String SHOW_DATABASES_SQL = "SHOW DATABASES ";
+
+  private static TsBlockSerde serde = new TsBlockSerde();
+
+  public IoTDBAbstractDatabaseMetadata(
+      IoTDBConnection connection, IClientRPCService.Iface client, long sessionId, ZoneId zoneId) {
+    this.connection = connection;
+    this.client = client;
+    this.sessionId = sessionId;
+    this.zoneId = zoneId;
+  }
+
+  static {
+    String[] allIotdbSQLKeywords = {
```

Comment:
keywords of table and tree model are different, you can find keywords of table model in https://timechor.feishu.cn/docx/U5OHdYQs1oR0hVxvy5Ucw0KUnBf